gradle.properties

```diff
@@ -1,4 +1,4 @@
-releaseVersion=0.8.0-rc8
+releaseVersion=0.8.0-rc8-local
```

Comment:
Is this a leftover or why is it called local now?

Reply:
a leftover, removing it